linera-service/src/node_service.rs

```diff
@@ -106,6 +107,7 @@ impl IntoResponse for NodeServiceError {
             | NodeServiceError::UnsupportedQueryType => (StatusCode::BAD_REQUEST, self.to_string()),
             NodeServiceError::GraphQLParseError { error } => (StatusCode::BAD_REQUEST, error),
         };
+        let tuple = (tuple.0, json!({"error": tuple.1}).to_string());
```

Comment:
Maybe `let (code, error) = match self {` in line 96 would make this easier to read.